src/studio-header/CourseLockUp.jsx

```diff
@@ -26,6 +29,7 @@ const CourseLockUp = ({
     <a
       className="course-title-lockup mr-2"
       href={outlineLink}
+      onClick={(e) => navigateToUrl(e, outlineLink, onNavigate)}
```

Comment:
Why add this `onClick` / `onNavigate` instead of using react-router's `<Link>` component?

Reply:
Implemented in follow-up commit: [`d5745e2` (#542)](https://github.com/openedx/frontend-component-header/pull/542/commits/d5745e210a9e7a57bb51ca229f4eebd00b365501)